packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -39,6 +41,11 @@ export const pollAppLogs = async ({
     },
   })
 
+  if (response.status === 401) {
+    await resubscribeCallback()
+    return
+  }
```

Comment:
>since the callback ends up going through the whole flow again (subscribe -> poll app logs -> setTimeout to call -> poll app logs -> setTimeout to call -> ...), at what point does this stop awaiting so that it can return here?

In a normal scenario, it'll stop waiting after the first successful poll